tests/python_client/testcases/test_delete.py

```diff
@@ -1920,6 +1921,7 @@ def test_delete_string_expressions_like(self, field_name, like, enable_dynamic_f
             data[ct.default_json_field_name] = [{"string": string_list[i]} for i in range(nb)]
             data[ct.default_string_field_name] = string_list
         collection_w.insert(data)
+        collection_w.flush()
```

Comment:
so here means delete sometime failed because it is not visible when deleting these entities?

Reply:
yes. Complex expression delete needs query first.